src/host/polyfills/releaseNotes.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export function applyReleaseNotePatch(content: string) {

```

Comment:
can we move this to it's own file? that way the changes are contained to that file and anybody can update the release nots without knowing about the polyfills

Reply:
Moved to `releaseNoteContents.ts`